Src/Extern/Conduit/AMReX_Conduit_Blueprint_ParticlesI.H

```diff
@@ -32,7 +31,7 @@ ParticleTileToBlueprint(const ParticleTile<amrex::Particle<NStructReal,
                         const std::string &topology_name)
 {
     int num_particles = ptile.GetArrayOfStructs().size();
-    int struct_size   = sizeof(Particle<NStructReal, NStructInt>);
+    int struct_size   = ParticleType::is_soa_particle ? 0 : sizeof(ParticleType);
```

Comment:
`struct_size` is used to represent element striding in the Conduit tree. 

For the SOA case - I expect we want native striding (for example, 8 bytes for float64), so I think we need to update the Particle Container to Conduit wrappers to handle these cases.

Reply:
Thank you Cyrus - where is that wrapper located? I thought it was this file...